cocos/ui/UIWebView.h

```diff
@@ -216,6 +216,20 @@ class CC_GUI_DLL WebView : public cocos2d::ui::Widget {
      * Toggle visibility of WebView.
      */
     virtual void setVisible(bool visible) override;
+    /**
+     * SetOpacity of webview.
+     */
+    virtual void setOpacityWebView(float opacity);
+    
+    /**
+     * getOpacity of webview.
+     */
+    virtual float getOpacityWebView();
```

Comment:
i think it is better to add const if possible, such as 
`virtual float getOpacityWebView() const`